source/jormungandr/jormungandr/external_services/external_service_provider_manager.py

```diff
@@ -171,6 +171,18 @@ def manage_free_floatings(self, navitia_service, arguments):
         # Return empty object instead of None if error occurs while calling external service.
         return service.get_response(arguments) if service else {'free_floatings': []}
 
+    # Here comes the function to call forseti/obstacles
+    def manage_obstacles(self, navitia_service, arguments):
```

Comment:
since the function is named explicitly "obstacles", you don't have to pass `navitia_service_type`
```suggestion
    def manage_obstacles(self, arguments):
```